source/bootcamp_content/open-source.rst

```diff
@@ -0,0 +1,122 @@
+Contributing to Open Source
+===========================
+
+Why get involved?
+-----------------
+
+* You can learn lots of new things, and grow as a developer
+* Meet amazing people
+* Give back to a community that has given you something
+* Personal fulfillment
+* You have more to contribute than you may realize!
+
+Know your licenses
```

Comment:
Licenses, as interesting as I think they are, should probably be later in the presentation. I know a lot of people check out when legal stuff gets brought up in any context. Perhaps after ``Finding a Project`` have something about ``Starting Your Own Project / Contributing With Others`` and the the license stuffs.